README.md

```diff
@@ -856,6 +857,14 @@ _Games, game servers and control panels._
 - [Sshwifty](https://github.com/nirui/sshwifty) - Sshwifty is a SSH and Telnet connector made for the Web. `AGPL-3.0` `Go/Docker`
 - [tmate](https://tmate.io/) - Instant terminal sharing. ([Source Code](https://github.com/tmate-io/tmate)) `ISC` `C`
 
+## Genealogy
+
+**[`^        back to top        ^`](#)**
+
+- [Genea.app](https://genea.app/) - Genea is a privacy by design and open source tool anyone can use to author or edit their family tree. Data is stored in the GEDCOM format and all processing is done in the browser. ([Demo](https://genea.app/), [Source Code](https://github.com/genea-app/genea-app)) `MIT` `Javascript`
```

Comment:
please remove the duplicate demo link

Reply:
Thank you for noticing, updated the record @nodiscc